certified-connectors/WorkSpan/apiProperties.json

```diff
@@ -124,58 +124,62 @@
       }
     },
     "iconBrandColor": "#c5dcff",
-    "publisher": "WorkSpan",
+	  "publisher": "WorkSpan",
```

Comment:
@WorkSpan-Engineering Please adjust the spacing for the `"publisher"` property.

Reply:
ok